packages/core/content-manager/admin/src/pages/ListView/ListViewPage.tsx

```diff
@@ -170,7 +174,7 @@ const ListViewPage = () => {
     return formattedHeaders;
   }, [displayedHeaders, formatMessage, list, runHookWaterfall, schema?.options?.draftAndPublish]);
 
-  if (isFetching) {
+  if (isLoading) {
```

Comment:
Is this right? I thought we could have refetching on listView and for that reason start using it

Reply:
I prefered using `currentData` from rtk so I get and empty array instead of a loading page everytime we refetch.

Reply:
I think it's more clear for the user to show him a loading screen than an empty view for a second. We definitely have too many spinners in Strapi, but IMO it should be fixed elsewhere (maybe react router loaders)

Reply:
it should just keep the old data until it can change the view but there are too many other issues to fix to get there for now. I don't have a preference I'll put back the full page loading if you prefer